scrapy/pipelines/images.py

```diff
@@ -128,19 +138,17 @@ def get_images(self, response, request, info):
             yield thumb_path, thumb_image, thumb_buf
 
     def convert_image(self, image, size=None):
-        if image.format == 'PNG' and image.mode == 'RGBA':
```

Comment:
Why is this part completely removed? Is the behavior the same (as before your patch) without it?